recipes/pysteps/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pysteps" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a2443ed3cd1636e60622a320cf7e7e8a0e857ad456e034ef19d084545ad1bc0e
+
+build:
+  number: 0
+  skip: True  # [py<36]"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - numpy
+    - cython
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - cython
+    - pip
+    - openmp 4.0.*  # [osx]
+    - attrdict
+    - jsmin
+    - scipy
+    - matplotlib
+    - jsonschema
+  run:
+    - python
+    - openmp 4.0.*  # [osx]
+    - attrdict
+    - jsmin
+    - jsonschema
+    - matplotlib
+    - netCDF4
+    - numpy
+    - opencv
+    - pillow
+    - pyproj
+    - scipy
+
+test:
+  imports:
+    - pysteps
+
+about:
+  home: https://pysteps.github.io/
+  license: BSD-3-Clause
+  license_family: BSD
+  summary: 'Python framework for short-term ensemble prediction systems'
+  license_file:
```

Comment:
In accordance with the terms of BSD-3-Clause, you need to specify a filename here. Ideally it should be included in the source tarball but if not you can include it in the recipe directory.

Reply:
Added in commit f4d208f , the LICENSE file is included in the sources.